### PR TITLE
Around comments

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: static/css/around-map.css [hunk 1]
```diff
@@ -38,7 +38,7 @@ main {
   display: flex;
   flex-direction: column;
   width: 49%;
-  padding: 2rem;
+  padding: 1rem 1.5rem 2.5rem;
```
Comment (vinyll) [inline static/css/around-map.css]: 0 for left? Why is it not the same padding all around the box?

File: static/css/around-map.css [hunk 2]
```diff
@@ -52,32 +52,33 @@ main {
...
-}
-
-.card h3 {
-  font-size: 1.85rem;
-  font-weight: normal;
-  margin: 0 1.25rem 0 0;
+  font-size: 1.7rem;
+  margin: .75rem 0;
+  max-height: 5rem;
+  width: 27rem;
```
Comment (vinyll) [inline static/css/around-map.css]: Fixed width for any size? `100%` wouldn't do the job?
Flexbox doesn't offer anything?

File: static/css/around-map.css [hunk 3]
```diff
@@ -52,32 +52,33 @@ main {
...
-  padding: 0 0 1rem 0;
+  list-style-type: none;
 }
 
-.comments {
-  flex-direction: column;
-  padding: 0;
+.card ul li div {
+  display: flex;
+  padding: 0 0 1rem 0;
```
Comment (vinyll) [inline static/css/around-map.css]: `padding-bottom: 1rem`?

File: static/css/around-map.css [hunk 4]
```diff
@@ -52,32 +52,33 @@ main {
...
 }
 
-.comments p {
-  margin: 1rem 0 0 0;
+.card ul li div h3 {
+  font-size: 1.85rem;
+  font-weight: normal;
+  margin: 0 1.25rem 0 0;
```
Comment (vinyll) [inline static/css/around-map.css]: `margin-right: 1.25rem`?

Comment (fuentesloic) [reply static/css/around-map.css]: h3 native comportement

**Review by vinyll — APPROVED** [review]

**Pull request merged**